docs/en/observability/grant-cases-access.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[grant-cases-access]]
+= Configure access to cases
+
+To access and send cases to external systems, you need the https://www.elastic.co/subscriptions[appropriate license],
+and your role must have the *Cases* {kib} privileges as a user for the *Observability* feature.
+
+Here are the minimum required privileges.
```

Comment:
Would/should we use a colon at end of sentence here?

Reply:
Added a colon. Thanks.